ui/src/components/Configuration/RequisitionDefinitionsLayout.vue

```diff
@@ -1,7 +1,12 @@
 <template>
   <div class="container">
     <div class="btnAction">
-      <Button :icon="icon" :label="buttonAction[index]" @click="clickAction(index)"></Button>
+      <FeatherButton primary @click="clickAction(index)">
```

Comment:
Is this really how featherds wants us to use a simple Icon Button? I wonder if it's worth us wrapping this functionality in a custom `<FeatherIconButton />`. component so we don't have to copy/paste the boilerplate with template/aria/focusable every time we just want an icon button. If their implementation ever changes it could be frustrating to go and replace every instance instead of just in one place.